AppBuilder/platform/dataFields/ABFieldConnect.js

```diff
@@ -637,9 +637,28 @@ module.exports = class ABFieldConnect extends ABFieldConnectCore {
          this.populateOptionsDataCy(theEditor, field, form);
       }
       if (theEditor.getValue?.() && data?.length) {
-         theEditor.setValue(theEditor.getValue());
-      }
+         let selectedVal = theEditor.getValue();
+
+         // Check exists item
+         const isExists = data.filter((d) => d.id == selectedVal).length > 0;
```

Comment:
suggest:
```js
data.some((d) => d.id == selectedVal);
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
Thank you.